React-Homeworks/src/components/Footer/Footer.jsx

```diff
@@ -1,8 +1,28 @@
-import "./styles.css"
-function Footer({ elements }) {
+import Navigation from "./Navigation/Navigation";
+import SocialMedia from "./SocialMedia/SocialMedia";
+import Copyright from "./Copyright/Copyright";
+import Links from "./SocialMedia/Links";
+import instagram from "../../assets/instagram.svg"
+import twitter from "../../assets/twitter.svg"
+import youtube from "../../assets/youtube.svg"
+import "./Footer.css"
+function Footer() {
     return (
-        <footer className="footer flex-elem">{elements}</footer>
+        <footer className="footer flex-elem">
+            <div className="footer__info flex-elem">
+            <Copyright/>
+            <Navigation/>
+            </div>
+            <div className="footer__media flex-elem">
+            <SocialMedia/>
+            <nav className="footer__nav flex-elem">
```

Comment:
In case you use tag nav it should be links inside and better wrapped into some list tag.